eng/build.ps1

```diff
@@ -22,7 +22,7 @@ $defaultargs = "-restore -build -warnaserror:0"
 $possibleDirToBuild = if($ExtraArgs.Length -gt 0) { $ExtraArgs[0]; } else { $null }
 
 if ($possibleDirToBuild -ne $null) {
-  $dtb = $possibleDirToBuild
+  $dtb = $possibleDirToBuild.TrimEnd('\')
```

Comment:
the else statement uses $possibleDirToBuild so it will not be trimmed still. You need to change it to join the path with $dtb instead. 😝

Reply:
😖😳😲😭😤

Reply:
thanks 👌😉